composer.json

```
@@ -20,25 +22,25 @@
         "erusev/parsedown": "^1.6",
         "sensio/framework-extra-bundle": "^6.1",
```

Bump other dependencies to versions that allow Symfony 6
```suggestion
        "sensio/framework-extra-bundle": "^6.2",
```